functions/methods.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+
+def read_the_file(file_path: str):
+    with open(file_path, 'r') as file:
+        data = file.readlines()
+        if data:
+            print('file_content:')
+            for line in data:
+                process_the_content(line.strip())
+        else:
+            print('file_is_empty.')
+
+
+def calculate_average(scores: List[float]):
```

Comment:
There is no need to import common collections from `typing` - you can use directly `list[float]` here